Joesdev/Exposure-API
# SD-29 Get Actions

## Whats This PR Do? 
- Adds Two Routes

1. **/api/action/{id}** Gets a single Action
2. **/api/hierarchy/{id}/actions** Gets All Actions In a Hierarchy
- Adds Two Functions

1. **HierarchyController@actions()**
2. **ActionController@show()**

- Adds test functions
## Steps To Review
- _There is no need to login to the application, also no need to run any database related _commands__
- **Navigate to /api/action/1** _Retrieves a single action whos id is 1_
- Verify data is returned
- **Navigate to /api/hierarchy/1/actions** Retreives all actions from a hierarchy whos id is 1
- Verify Data is returned
